src/__experimental__/components/radio-button/radio-button.stories.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import {
+  boolean, text, number, select
+} from '@storybook/addon-knobs';
+import OptionsHelper from '../../../utils/helpers/options-helper';
+import { RadioButton, RadioButtonGroup } from '.';
+import { info, notes } from './documentation';
+
+storiesOf('Experimental/RadioButton', module)
+  .add('default', () => {
+    return (
+      <RadioButtonGroup
+        groupName='frequenct'
+        label={ text('groupLabel', 'Please select a frequency from the options below') }
+      >
+        <RadioButton
+          id={ text('radioOneId', 'input-1') }
+          label={ text('radioOneLabel', 'Example Weekly Radio Button') }
+          value={ text('radioOneValue', 'weekly') }
+          { ...defaultKnobs() }
```

Comment:
Due to the way `@storybook/addon-knobs` works, this story's knobs are currently displayed in this order in Storybook (with `radioOne*` separated from `radioTwo*` and `radioThree*`):
```
groupLabel
radioOneId
radioOneLabel
radioOneValue
disabled
error
fieldHelp
fieldHelpInline
reverse
labelHelp
inputWidth
labelWidth
labelAlign
size
radioTwoId
radioTwoLabel
radioTwoValue
radioThreeId
radioThreeLabel
radioThreeValue
```
By applying a patch like this:
```diff
 storiesOf('Experimental/RadioButton', module)
   .add('default', () => {
+    const knobs = defaultKnobs();
     return (
       <RadioButtonGroup
         groupName='example'
         label={ text('groupLabel', 'Select an option from the radio buttons below') }
       >
         <RadioButton
           id={ text('radioOneId', 'input-1') }
           label={ text('radioOneLabel', 'Example radio button 1') }
           value={ text('radioOneValue', 'test-1') }
-          { ...defaultKnobs() }
+          { ...knobs }
         />
         <RadioButton
           id={ text('radioTwoId', 'input-2') }
           label={ text('radioTwoLabel', 'Example radio button 2') }
           value={ text('radioTwoValue', 'test-2') }
-          { ...defaultKnobs() }
+          { ...knobs }
         />
         <RadioButton
           id={ text('radioThreeId', 'input-3') }
           label={ text('radioThreeLabel', 'Example radio button 3') }
           value={ text('radioThreeValue', 'test-3') }
-          { ...defaultKnobs() }
+          { ...knobs }
         />
       </RadioButtonGroup>
     );
   }, {
```
then the knobs will be displayed in this order (with `radioOne*` next to `radioTwo*` and `radioThree*`, which is more user-friendly):
```
disabled
error
fieldHelp
fieldHelpInline
reverse
labelHelp
inputWidth
labelWidth
labelAlign
size
groupLabel
radioOneId
radioOneLabel
radioOneValue
radioTwoId
radioTwoLabel
radioTwoValue
radioThreeId
radioThreeLabel
radioThreeValue
```

Reply:
Thanks! 🙏 

I knew there was a way I could sort this out, but couldn't remember what to do, and then forgot to go back to it 😂